rero_ils/modules/selfcheck/api.py

```diff
@@ -503,7 +520,7 @@ def selfcheck_checkin(transaction_user_pid, item_barcode, **kwargs):
                     _('No circulation action is possible'))
             except CirculationException as circ_err:
                 checkin.get('screen_messages', []).append(
-                    _(circ_err.description))
+                    circ_err.description)
```

Comment:
Why no translation for `circ_err.description`? 
(https://github.com/pallets/werkzeug/blob/main/src/werkzeug/exceptions.py)